gh-deployments-environments/helper.sh

```diff
@@ -0,0 +1,201 @@
+#!/usr/bin/env bash
+
+set -e
```

Comment:
I would go with `set -eu` at least, this `-u` is very helpful with preventing bash error when it comes to passing envs to scripts

Reply:
Fixed.  I'll resolve this conversation, it's approved and I'd like to use that tiny github action already in one tiny workflow we have that tidies up featurenets.